andela/ah-baqi-frontend
# #166980212 Rating articles

### What does this PR do?
Adds Rating Articles Functionality
### Description of Task to be completed?
   - [x] add five-star rating 
### How should this be manually tested?
- Checkout to:

      ft-rate-article-166980212
     
- Install dependencies:

      yarn
     
- Run local server:

      yarn develop

- Click on rating stars at the footer of the article
- Confirm your rating by making sure that the rate average on the artcile header has increased or decreased
### Any background context you want to provide?
None
### What are the relevant pivotal tracker stories?
#166980212
### Screenshots (if appropriate)
<img width="1440" alt="Screenshot 2019-07-19 at 07 17 42" src="https://user-images.githubusercontent.com/42172425/61509021-546caa00-a9f5-11e9-94fb-909cbf7fa7bb.png">
